docs/_pages/docs/03-understand/05-trade-escrow.md

```diff
@@ -2,10 +2,60 @@
+Be sure to use a Lightning wallet that plays well with RoboSats, refer to [Understand > Wallets](https://learn.robosats.com/docs/wallets/) for additional information.
+
+*Note: The term "seller" specifically refers to selling bitcoin while "buyer" specifically refers to buying bitcoin.*
+
+## **How to Submit a Trade Escrow**
+
+First, refer to [Understand > Wallets](https://learn.robosats.com/docs/wallets/) for compatible Lightning wallets that will help make using RoboSats a smoother experience. Depending on the wallet, the locked funds might show as a payment that is in transit, frozen, or even appearing to fail. Check the wallet compatability list!
+
+Read the relevant guide depending on if you are making or taking an order to sell bitcoin:
+* **Maker**: Select "Make Order" and modify the order conditions to your liking. The order can be customized to require an "Escrow Deposit Time-Out" (expiry timer) other than the default of three hours, ranging anywhere from one to eight hours. When your live order gets taken and the taker submits their payout invoice, use the shown QR code found in the "Contract Box" with your Lightning wallet to lock the indicated amount of sats as collateral (escrow). You can now chat with your robot peer to coordinate the fiat payment. *Note: Escrow funds are released to the buyer once you select "Confirm Fiat Received" which settles the order. Only confirm after the fiat has arrived in your account.*
+* **Taker**: Browse the order book and find an order to your liking. Simply select the "Take Order" option and lock your fidelity bond. Only after the buyer submits their payout invoice will you be required to submit the trade escrow. Use the shown QR code found in the "Contract Box" with your Lightning wallet to lock the indicated amount of sats as collateral (escrow). You can now chat with your robot peer to coordinate the fiat payment. *Note: Escrow funds are released to the buyer once you select "Confirm Fiat Received" which settles the order. Only confirm after the fiat has arrived in your account.*
+
+The escrow is not required by the seller until the buyer has locked their payout invoice. As soon as the buyer submits the invoice, the seller then has three hours (by default) to lock the escrow. If the timer runs out, then the order expires and the seller forfeits their bond.
+
+By default, the expiry timer for the escrow is three hours; however, you can customize the order to have an "Escrow Deposit Time-Out" ranging from one hour to eight hours. In other words, modify the time allowed for the seller to submit the escrow funds. Maybe you want a quicker transaction and will set the timer to a maximum of one hour instead of three hours.
+
+Once the fiat payment method is coordinated with your fellow robot, the seller selects "Confirm Fiat Received" to end the trade which releases the escrow funds to the buyer. Only confirm the fiat is received *after* it appears in your possession.
+
+## **Losing Your Trade Escrow**
```

Comment:
`** Losing Your Trade Escrow**` I would change it to `How and when the escrow is released to the buyer`